cmd/weirproxy/main.go

```diff
@@ -73,9 +73,9 @@ func main() {
 	)
 
 	var wg sync.WaitGroup
-	wg.Add(1)
 
 	go func() {
+		wg.Add(1)
```

Comment:
Why move it here?

Reply:
No special intention. I think, because I copied `waitGroupWrapper` from hawkingrei, and his wrapper utils are doing so. Thus I made such changes.

Reply:
I could change it back. It doesn't matter too much.

Anyway, I have a big refactor PR to introduce etcd/cobra, and it makes dependencies much simpler.

Reply:
You mean because of some scheduling problem, so it should be outside of the vo routine. Sounds good to me, I:ll change it.